### DISCUSSION TITLE
Portable Version of MTB

### DISCUSSION THREAD

**Discussion by mff73** [mytourbook/mytourbook#1073] [General]
Hi,
i am using MTB on my professional laptop, and security was enforced and no updates/installation are now possible :(
Is there already a way to run MTB like a portable version (meaning from a usb key or disk = without any installation ) ?
My db is already on a remote storage, but main program is on laptop.

PS: i am also using my personal computer, so i can eventually make an install there and then copy new versions, if one tell me what to copy in which way 😊

If nothing possible, then, is there a way to create a real portable version ?

thanks

**Comment by Thompson-ongithub** [comment]
Hi mff73
I'm not entirely sure since I'm using Linux, but isn't the Windows zip-Version portable (you might need to have Java installed I admit)?
As for the db you seem to know already how to redirect that....
I hope this helped...

**Comment by mff73** [comment]
Hi @Thompson-ongithub ,
very good idea indeed, and it seems to work :) I am running 23.3 version, while only 22.11 is installed ;)
Thanks.
